### PR TITLE
add api call for /project/{projectIdOrKey}/version

### PR DESCRIPTION
**Pull request by gierappa**

### CODE REVIEW

**Comment by MrRefactoring** [comment]
Looks good to pour these changes.

Can you add typings for support `typescript`?

File: api/project.js [hunk 1]
```diff
@@ -184,6 +184,28 @@ function ProjectClient(jiraClient) {
...
+     * @param [callback] Called when the paginated representation of all versions have been retrieved.
+     * @return {Promise} Resolved when the paginated representation of all versions have been retrieved.
+     */
+    this.getVersionsPaginated = function (opts, callback) {
+        var qs = {
+            startAt: opts.startAt,
+            maxResults: opts.maxResults,
+            orderBy: opts.orderBy,
+            query: opts.query,
+            status: opts.status
```
Comment (MrRefactoring) [inline api/project.js]: I think you forgot to add `expand` property

File: api/project.js [hunk 2]
```diff
@@ -184,6 +184,28 @@ function ProjectClient(jiraClient) {
         return this.jiraClient.makeRequest(options, callback);
     };
 
+    /**
+     * Contains a paginated representation of a the specified project's versions.
+     *
+     * @method getVersionsPaginated
+     * @memberOf ProjectClient#
+     * @param opts The request options sent to the Jira API.
+     * @param opts.projectIdOrKey The project id or project key
```
Comment (MrRefactoring) [inline api/project.js]: Please add missed params (`startAt`, `maxResults`, `orderBy`, `query`, `status`, `expand`) and define types for it

File: api/project.js [hunk 3]
```diff
@@ -184,6 +184,28 @@ function ProjectClient(jiraClient) {
...
+     * @param [callback] Called when the paginated representation of all versions have been retrieved.
```
Comment (MrRefactoring) [inline api/project.js]: Also, can you define `{callback}` type for `[callback]`?

```js
* @param {callback} [callback]
```

File: api/project.js (hunk 1 above)
Comment (gierappa) [reply api/project.js]: exapand is added automagically

File: api/project.js (hunk 3 above)
Comment (gierappa) [reply api/project.js]: Done.

File: api/project.js (hunk 1 above)
Comment (MrRefactoring) [reply api/project.js]: Yeah see thanks!

**Comment by MrRefactoring** [comment]
@gierappa thank you for the contribution! We add these changes to NPM in the next release!

**Comment by gierappa** [comment]
@MrRefactoring don't forget to update readme with new call:) cheers

**Comment by MrRefactoring** [comment]
> @MrRefactoring don't forget to update readme with new call:) cheers

Good point. Сompletely forgot)

**Pull request merged**